src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -490,7 +490,8 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
            call mpas_pool_get_array(diagnosticsPool, 'RiTopOfCell', RiTopOfCell)
            call mpas_pool_get_array(diagnosticsPool, 'zTop', zTop)
            call mpas_pool_get_array(diagnosticsPool, 'c_visbeck', c_visbeck)
-           !$omp do schedule(runtime) private(cell1, cell2, sumN2, ltSum, sumRi, countN2, zEdge, RiTopOfEdge, BruntVaisalaFreqTopEdge) 
+           !$omp do schedule(runtime) private(k, cell1, cell2, sumN2, ltSum, sumRi, countN2, &
+           !$omp& zEdge, RiTopOfEdge, BruntVaisalaFreqTopEdge)
```

Comment:
@mark-petersen - I think you need to add k to the omp pragmas at 424 and 452 above as well?